src/EbmlElement.cpp

```diff
@@ -714,14 +714,33 @@ filepos_t EbmlElement::OverwriteHead(IOCallback & output, bool bKeepPosition)
   return Result;
 }
 
+filepos_t EbmlElement::OverwriteData(IOCallback & output, bool bKeepPosition)
+{
+	if (ElementPosition == 0) {
+		return 0; // the element has not been written
+	}
+
+	uint64 HeaderSize = EbmlId(*this).GetLength() + CodedSizeLength(Size, SizeLength, bSizeIsFinite);
+
+	filepos_t DataSize = GetSize();
+
+	uint64 CurrentPosition = output.getFilePointer();
+	output.setFilePointer(GetElementPosition() + HeaderSize);
+	filepos_t Result = RenderData(output, true, bKeepPosition);
+	output.setFilePointer(CurrentPosition);
```

Comment:
Forcefully using `setFilePointer` without checking `bKeepPosition` doesn't make much sense to me.

Reply:
I'm not sure what you're requesting here.  This function has a very similar flow to ```OverwriteHead```, with only a single line difference (compare lines 712 and 729).